src/commands/coverage_publish.yml

```diff
@@ -18,12 +18,20 @@ parameters:
       specified, the format will be inferred from the file
       extension or contents [possible values: simplecov, clover,
       cobertura, coverprofile, lcov, jacoco, qlty].
-    type: string
-    default: ""
+    enum:
+      - clover
+      - cobertura
+      - coverprofile
+      - jacoco
+      - lcov
+      - simplecov
+      - qlty
+    type: enum
```

Comment:
Note, I didn't specify a default because I couldn't find code attempting to infer the format.  Also, requiring a value seems appropriate for this param.  Thoughts?

Reply:
The CLI has a basic mechanism to try and infer the report format from the filename, so this parameter should be optional. Does that mean we need to add `default: ""`?

Reply:
That works.  Ill add an empty string option set to the default.